src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -414,9 +430,7 @@ UsersFragment.propTypes = {
   initialUsers: PropTypes.array,
   setInitialUsers: PropTypes.func,
   followedUsers: PropTypes.array,
-  setFollowedUsers: PropTypes.func,
   unfollowedUsers: PropTypes.array,
-  setUnfollowedUsers: PropTypes.func,
   setSearchText: PropTypes.func,
   setIsFirstTimeOpen: PropTypes.func,
   withoutRecent: PropTypes.bool,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of PropTypes for `setFollowedUsers` and `setUnfollowedUsers` aligns with the changes made earlier in the code where these functions were removed from the props. This is consistent and indicates that these functions are no longer expected as props.


<!-- This is an auto-generated comment by OSS CodeRabbit -->